script.js

```diff
@@ -24,4 +24,156 @@ cancelButton.setAttribute('onclick', 'closeMenu()');
 
 for (let i = 0; i < 3; i += 1) {
   mobileLinks[i].setAttribute('onclick', 'closeMenu()');
-}
\ No newline at end of file
+}
+
+const popUpContent = [
```

Comment:
- [ ] Great job with the array popup contents. Please have in mind that each array item represents a separate project you want to display. Kindly add your items according to the number of cards you have(4)

Reply:
- [ ] As suggested by the previous reviewer, kindly make sure that each array item represents a separate project you want to display. Kindly add your items according to the number of cards you have(4).

This means you have to make 4 arrays for your project and then display them using their arrays.